shared/origami/include/origami/gemm.hpp

```diff
@@ -75,9 +75,9 @@ namespace origami
     // Computes the number of active compute units if there is only one wave and it is partial
     // Otherwise, returns hardware.N_CU
     size_t compute_active_CU(
```

Comment:
I think we can use `cu` instead of `CU` as well.

```suggestion
    size_t compute_active_cu(
```